locate_qualified_library search nonexact matches in 1 step (#20601)
rocq-prover/rocq: doc/changelog/09-cli-tools/20601-find-unique-Changed.rst

- **Added:**

>   error on ambiguous require. Rocq used to silently select a file

Reviewer: ```suggestion
  error on ambiguous :cmd:`Require`. Rocq used to silently select a file
```

>   when ambiguous requires came from different loadpaths, for instance

Reviewer: ```suggestion
  when ambiguous :cmd:`Require`\s came from different loadpaths, for instance
```

>   diferent fields of the ``ROCQPATH`` environment variable

Reviewer: ```suggestion
  different fields of the ``ROCQPATH`` environment variable
```
